packages/dds/tree/README.md

```diff
@@ -46,8 +46,6 @@ To access the `beta` APIs, import via `@fluidframework/tree/beta`.
 Notable consideration that early adopters should be wary of:
 
 -   The persisted format is stable such that documents created with released versions 2.0.0 or greater of this package are fully supported long term.
--   SharedTree currently has unbounded memory growth:
-    -   Removed content is retained in memory and persisted in the document at rest ([fix](https://github.com/microsoft/FluidFramework/pull/21372)).
```

Comment:
Rather than remove this, I would leave it and instead update it to say something like, "In versions prior to 2.1.0, SharedTree had unbounded memory growth, and removed content... This was fixed in version 2.1.0."